test/test_other.py

```diff
@@ -9713,7 +9733,7 @@ def test_ioctl(self):
     # ioctl requires filesystem
     self.do_other_test('test_ioctl.c', emcc_args=['-sFORCE_FILESYSTEM'])
 
-  @also_with_noderawfs
+  # @also_with_noderawfs # NODERAWFS doesn't use library_tty.js for stdio streams
```

Comment:
It seems that this should probably work under NODERAWFS.   Perhaps open a bug an refer it that bug here?

Reply:
Changed to: `# @also_with_noderawfs # NODERAWFS reports ENOTTY, see issue #22264.` with commit faac3bc62820b63205890b91ac513b0c035d71b1 instead.